packages/web/src/server/index.ts

```diff
@@ -32,24 +31,35 @@ interface RequestContext {
 }
 
 const symbol = Symbol.for('@vercel/request-context');
+const logPrefix = '[Vercel Web Analytics]';
 
 export async function track(
   eventName: string,
   properties?: Record<string, AllowedPropertyValues>,
   context?: Context
 ): Promise<void> {
+  if (typeof window !== 'undefined') {
+    if (!isProduction()) {
+      throw new Error(
+        `${logPrefix} It seems like you imported the \`track\` function from \`@vercel/web-analytics/server\` in a browser environment. This function is only meant to be used in a serverless environment.`
```

Comment:
I believe edge is also working, maybe we should only mention `server` instead of `serverless`.

```suggestion
        `${logPrefix} It seems like you imported the \`track\` function from \`@vercel/web-analytics/server\` in a browser environment. This function is only meant to be used in a server environment.`
```